website_portal_debt_notebook/static/src/js/test_website_portal_debt_notebook.js

```diff
@@ -0,0 +1,42 @@
+/*  Copyright 2019 Anvar Kildebekov <https://it-projects.info/team/fedoranvar>
+    License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html). */
+odoo.define('website_portal_debt_notebook.tour', function (require) {
+    "use strict";
+
+    var tour = require("web_tour.tour");
+    var core = require('web.core');
+    var base = require("web_editor.base");
+    var _t = core._t;
+
+    function check_debt_history(journal, cash) {
+        return [{
+            trigger: "a[href*='/my/debt_history']:first",
+            content: _t("<p>Click to enter 'debt history'</p>"),
+            position: "bottom",
+        },{
+            trigger: ".history-line:nth-child(2)",
+            content: _t("<p>Check credit-lines</p>"),
+            position: "bottom",
+            run: function(){
+                for (var i = 0; i < 2; i++) {
+                   if ($(".history-line:nth-child(" + (i+1).toString() + ")>td:nth-child(2)")[0].innerText !== journal ||
+                       $(".history-line:nth-child(" + (i+1).toString() + ")>td:nth-child(4)")[0].innerText !== cash[i]) {
+                        console.error(i + '-line failed: ' +
```

Comment:
not sure it is going to throw a proper error.
read the description here https://github.com/odoo/odoo/blob/12.0/odoo/tests/common.py#L907